backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/serialization/KonanIrModuleSerializer.kt

```diff
@@ -21,7 +21,8 @@ private class KonanDeclarationTable(
      */
     override fun tryComputeBackendSpecificUniqId(declaration: IrDeclaration): UniqId? {
         return if (declaration.descriptor.module.isFromInteropLibrary()) {
-            UniqId(declaration.descriptor.getUniqId() ?: error("No uniq id found for ${declaration.descriptor}"))
+            // Property accessor doesn't provide UniqId so we need to get it from the property itself.
```

Comment:
Why so? How does this work for standard IR libraries?

Reply:
In case of IR-based libraries, we compute UniqId from `IrDeclaration`. For IR-less libraries we use pre-calculated UniqId value that is stored in metadata. Since there is no metadata for property accessors, we have to use the one from the property.